report/test-assets/faux-psi.js

```diff
@@ -82,6 +87,22 @@ async function renderLHReport() {
   }
 }
 
+/**
+ * @param {LH.Locale} locale
+ */
+async function swapLhrLocale(locale) {
+  const response = await fetch(`https://www.gstatic.com/pagespeed/insights/ui/locales/${locale}.json`);
```

Comment:
In the real thing, is the list of locales suggested in the UI generated from the actual locales available so it's not going to 404?

Reply:
ideally, but there's no certainty the locales obj we have matches what's published here. but.. it _should_.

I think what you may be pointing out is that we should do a `lookupLocale` if the locale code coming from PSI hasnt already underwent that.  At this point, I'm not sure, but i'll find out. Easier to find out once we can try out this PR. :)

Reply:
> flagging this comment
> 
> [#13885 (comment)](https://github.com/GoogleChrome/lighthouse/pull/13885#discussion_r859063448)

addressed. @connorjclark  ^